### DISCUSSION TITLE
Why don't the releases include a changelog anymore?

### DISCUSSION THREAD

**Discussion by TheOnlyTails** [JanDeDobbeleer/oh-my-posh#1343] [Troubleshoot] [answered]
title

**Comment by JanDeDobbeleer** [comment, marked as answer]
Because it seems I made a typo. Fixed.

**Reply by TheOnlyTails** [reply]
Wonderful!
It might be worth it to keep a log of all the changelogs in something like `CHANGELOG.md`

**Reply by JanDeDobbeleer** [reply]
> Wonderful!
> 
> It might be worth it to keep a log of all the changelogs in something like `CHANGELOG.md`

No, that requires an additional meaningless commit. The whole context is in the commit messages. All you need is to read through the git history 😉

**Reply by TheOnlyTails** [reply]
OK then, having the changes in the releases is good too

**Reply by JanDeDobbeleer** [reply]
I'll see if I can regenerate the lost ones. That'll teach me for releasing so often.

**Reply by TheOnlyTails** [reply]
Thank you, that'd be great!
